Tests/NIOHTTP2Tests/SimpleClientServerFramePayloadStreamTests.swift

```diff
@@ -198,9 +198,14 @@ class SimpleClientServerFramePayloadStreamTests: XCTestCase {
     /// Establish a basic HTTP/2 connection.
     func basicHTTP2Connection(clientSettings: HTTP2Settings = nioDefaultSettings,
                               serverSettings: HTTP2Settings = nioDefaultSettings,
+                              maximumBufferedControlFrames: Int = 10000,
```

Comment:
The changes in this file makes these tests run faster: I was sick of waiting for them.